lib/Raisin/Encoder/JSON.pm

```diff
@@ -3,12 +3,16 @@ package Raisin::Encoder::JSON;
 use strict;
 use warnings;
 
-use JSON qw(encode_json decode_json);
+use JSON::MaybeXS  qw(encode_json decode_json );
+my $json=JSON::MaybeXS->new();
```

Comment:
And just minor styling issue, could you please put one space around `=`?

Reply:
done